magento/magento-cloud-docker
# MAGECLOUD-3979: [Cloud Docker] Exclude large files from docker-sync and mutagen

### Description


### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-3979


### Manual testing scenarios

1. Install magento on docker in development mode
2. Use docker-sync/mutagen for files sync.
3. Create some `.sql`/`.gz`/`.zip`/`.bz2` file in your project dir
4. Check that file wasn't sync with docker env. 

https://jira.corp.magento.com/browse/MAGECLOUD-115

### Contribution checklist
